{
    "issue": {
        "title": "[Rust] Intermittent Flight integration test failures",
        "body": "***Note**: This issue was originally created as [ARROW-11908](https://issues.apache.org/jira/browse/ARROW-11908). Please see the [migration documentation](https://gist.github.com/toddfarmer/12aa88361532d21902818a6044fda4c3) for further details.*\n\n### Original Issue Description:\nThis is similar to the symptoms seen in ARROW-11717 but it is still happening  intermittently\r\n\r\nOn two separate PR I see similar failures:\r\nhttps://github.com/apache/arrow/pull/9645/checks?check_run_id=2052183132\r\nhttps://github.com/apache/arrow/pull/9647/checks?check_run_id=2051946608\r\n\r\nExample failure:\r\n\r\n```Java\n\r\n\r\nsubprocess.CalledProcessError: Command '['/build/cpp/debug/flight-test-integration-client', '-host', 'localhost', '-port=41743', '-scenario', 'auth:basic_proto']' died with <Signals.SIGABRT: 6>.\r\n\r\nDuring handling of the above exception, another exception occurred:\r\n\r\nTraceback (most recent call last):\r\n  File \"/arrow/dev/archery/archery/integration/runner.py\", line 308, in _run_flight_test_case\r\n    consumer.flight_request(port, **client_args)\r\n  File \"/arrow/dev/archery/archery/integration/tester_cpp.py\", line 116, in flight_request\r\n    run_cmd(cmd)\r\n  File \"/arrow/dev/archery/archery/integration/util.py\", line 148, in run_cmd\r\n    raise RuntimeError(sio.getvalue())\r\nRuntimeError: Command failed: /build/cpp/debug/flight-test-integration-client -host localhost -port=41743 -scenario auth:basic_proto\r\nWith output:\r\n--------------\r\n-- Arrow Fatal Error --\r\nInvalid: Expected UNAUTHENTICATED but got Unavailable\r\n\r\n--------------\r\n\r\n################# FAILURES #################\r\nFAILED TEST: auth:basic_proto Rust producing,  C++ consuming\r\n\r\n```",
        "created_at": "2021-03-08T21:51:39.000Z",
        "updated_at": "2021-04-26T12:49:38.000Z",
        "labels": [
            "Migrated from Jira",
            "Component: Integration",
            "Component: Rust",
            "Type: bug"
        ],
        "closed": true,
        "closed_at": "2021-04-26T12:49:38.000Z"
    },
    "comments": [
        {
            "created_at": "2021-03-16T12:01:32.058Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-11908?focusedCommentId=17302453) by David Li (lidavidm):*\n`[~alamb]` \u00a0are you taking a look at this? Otherwise I'll add a retry on the C++ side."
        },
        {
            "created_at": "2021-03-22T14:09:37.405Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-11908?focusedCommentId=17306216) by Antoine Pitrou (apitrou):*\nIssue resolved by pull request 9755\n<https://github.com/apache/arrow/pull/9755>"
        },
        {
            "created_at": "2021-03-22T14:10:31.314Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-11908?focusedCommentId=17306217) by Antoine Pitrou (apitrou):*\nSorry for the spurious closing. `[~lidavidm]` 's PR is just a temporary workaround, but this should be cleanly fixed on the Rust side."
        },
        {
            "created_at": "2021-04-26T12:49:36.873Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-11908?focusedCommentId=17332233) by Andrew Lamb (alamb):*\nMigrated to github: https://github.com/apache/arrow-rs/issues/201"
        }
    ]
}